src/components/SearchBar.js

```diff
@@ -96,7 +114,12 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
     autoCompleteUri: null,
     route: null,
     baseUri: '',
+    shareSearch: false,
     createAutoCompleteSignal: false,
+    shareMessage: 'Hey,\n\nI think you would be interested in these search results that I found using the most'
```

Comment:
Prefer [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). They're easier to read.
```suggestion
    shareMessage: `Hey, I think you would be interested in these search results that I found using Attivio, a leader in cognitive search and knowledge discovery. Here is the link:`,
```